evm_loader/cli/src/main.rs

```diff
@@ -147,7 +148,19 @@ fn command_emulate(config: &Config, contract_id: H160, caller_id: H160, data: Ve
 
     let accounts: Vec<AccountJSON> = account_storage.get_used_accounts();
 
-    let js = json!({"accounts": accounts, "result": &hex::encode(&result), "exit_status": status}).to_string();
+    let account_metas: Vec<AccountMetaJSON> = account_storage.external_account_metas
```

Comment:
`AccountMeta` name is too ambiguous. Try to find a more accurate name.

Reply:
Fixed